src/main/java/seedu/address/logic/commands/WhitelistCommand.java

```diff
@@ -21,8 +21,11 @@ public class WhitelistCommand extends Command {
 
     public static final String COMMAND_WORD = "whitelist";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + " index cs/CLIENT_STATUS"
-            + ": removes person from the blacklist";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " INDEX cs/CLIENT_STATUS"
+            + ": removes person from the blacklist and sets their client status to CLIENT STATUS."
+            + " CLIENT STATUS must be 'old', 'active', 'potential' or 'unresponsive'."
+            + "\nExample: '" + COMMAND_WORD + " 2 cs/old' sets the client status of the 2nd "
+            + "client in the list as 'old' after removing them from the blacklist.";
```

Comment:
I notice some inconsistencies between the command usage message and the user guide:
<img width="438" alt="image" src="https://github.com/user-attachments/assets/503a667c-1096-4045-aab7-7a4611d627aa">

1.  User Guide Format shows "whitelist index cs/NEW_CLIENT_STATUS" but MESSAGE_USAGE uses:
   - "INDEX" (different case)
   - "CLIENT_STATUS" (missing "NEW_" prefix)
   - "CLIENT STATUS" (with space)

Could we align all these to consistently use "NEW_CLIENT_STATUS" as shown in the user guide?

Reply:
Sure, I can do that

Reply:
Thanks for the review!